File: Projects/po_projekt2/worldElements/Coordinates.java

package po_projekt2.worldElements;

/**
 *
 * @author Bogna
 */
public class Coordinates {
    private int x, y;
    
    public Coordinates(){
        this.x = 0;
        this.y = 0;
    }
    
    public Coordinates(int x, int y){
        this.x = x;
        this.y = y;
    }
    
    public int getX(){
        return x;
    }
    
    public int getY(){
        return y;
    }
    
    public void setX(int x){
        this.x = x;
    }
    
    public void setY(int y){
        this.y = y;
    }
}

File: Project_1-c++/Coordinates.cpp
#include "Coordinates.h"

Coordinates::Coordinates(int x, int y) {
	this->x = x;
	this->y = y;
}

Coordinates::Coordinates() {
	x = 0;
	y = 0;
}

int Coordinates::GetX() {
	return x;
}

int Coordinates::GetY() {
	return y;
}

void Coordinates::SetX(int x) {
	this->x = x;
}

void Coordinates::SetY(int y) {
	this->y = y;
}

Coordinates::~Coordinates() {}